x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPolicyRunner.java

```diff
@@ -381,14 +378,14 @@ private void prepareAndCreateEnrichIndex(List<Map<String, Object>> mappings) {
             // This disables eager global ordinals loading for all fields:
             .put("index.warmer.enabled", false)
             .build();
-        CreateIndexRequest createEnrichIndexRequest = new CreateIndexRequest(enrichIndexName, enrichIndexSettings);
+        CreateIndexRequest createEnrichIndexRequest = new CreateIndexRequest(targetIndexName, enrichIndexSettings);
```

Comment:
I think I actually prefer `enrichIndexName` over `targetIndexName` everywhere, do you have a preference?

Reply:
I'll swap it back over to enrichIndexName